Schmidt/template/README.md

```diff
@@ -1,13 +1,13 @@
-Ok so I guess you are reading this cuz you wanna use my code. There are some
-functions that do stuf and thats:
+
+factorial is a function that returns something important.
 
     >>> from simple_functions import factorial
     >>> factorial(10)
     9
 
-and this other part does something.  I forget why that I did it:
+The function fibonnacci returns a fibonacci sequence of all numbers that are less than the given numbers. 
 
     >>> fibonnaccci(100)
     [0, 1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89]
 
-If you can't use it, its kind of your problem, not mine!
+
```

Comment:
It's important we know who's hard work went into these changes, you should add yourself as author at the bottom of the readme.

Reply:
Done!